There were some issues with trying to import nltk via the remote SSH to a linux uleth computer. When I tried to run my program by adding the repo files to my account directory then tried "python3 a1.py" to run it, it gave me an error saying there is not such module for nltk. This made it so I was not able to actually properly run my python program via SSH, but managed to get it to work on Unbuntu for windows to confirm it should work in a linux environment. This was only due to the fact I was able to install pip3 to my unbuntu to go with python 3.8 in order to install nltk before running my program which resulted in it working properly.
